src/state/item/item-reducers.js

```diff
@@ -141,22 +142,25 @@ export default function (state = initialState, action) {
         }
 
         case itemActionTypes.DESCRIPTION_UPDATED: {
-            const { description, tempDescription } = action.data;
-            const { activeItem } = state;
+            const { description } = action.data;
+            const { activeItemEditing } = state;
             return {
                 ...state,
-                activeItem: {
-                    ...activeItem,
+                activeItemEditing: {
+                    ...activeItemEditing,
                     description,
                 },
-                tempDescription: tempDescription || state.tempDescription,
             };
         }
 
         case itemActionTypes.DESCRIPTION_UPDATE_SUCCEEDED: {
+            const { activeItem, activeItemEditing } = state;
             return {
                 ...state,
-                tempDescription: state.activeItem.description,
+                activeItem: {
+                    ...activeItem,
```

Comment:
This doesn't really do anything anymore because we are refreshing the document anyways.  Thoughts?  Make it match what `METADATA_UPDATE_SUCCEEDED` is doing since it also refreshes the document anyways?